tests/astrapy/test_db_dml.py

```diff
@@ -790,6 +790,26 @@ def test_update_one_create_subdocument_novector(
     assert response["data"]["document"]["name"] == "Eric"
 
 
+@pytest.mark.describe("update_many to create a subdocument, not through vector")
+def test_update_many_create_subdocument_novector(
+    writable_v_collection: AstraDBCollection,
```

Comment:
Replace with `empty_v_collection` fixture otherwise you may have surprises with the count at the end (writable gives you no guarantee there _won't_ be other records shadowing your failing test at the last assert).

Reply:
Thanks @hemidactylus . great point. Updated!